src/ducks/files/Toolbar.jsx

```diff
@@ -29,6 +30,8 @@ class Toolbar extends React.Component {
 
   render () {
     const { t, disabled, displayedFolder, actions, onSelectItemsClick, uploadFiles } = this.props
+    const notRootfolder = displayedFolder !== undefined && displayedFolder !== null && displayedFolder.dir_id === ROOT_DIR_ID ? true : false
```

Comment:
actually ROOT_DIR_ID works oddly, its value is `io.cozy.files.root-dir`and If you're in the root folder the value of `displayedFolder.dir_id` attribute is an empty string `""`, if you're in a level 1 folder it's `io.cozy.files.root-dir` and if if you're in a level 3 or more folder it's an hashed id. 
So right now I got back to the test of `displayedFolder.path` value which is a simpler test to do.